src/app/plugins/register.plugin.ts

```diff
@@ -17,16 +17,34 @@ export class RegisterPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args?: string[]) {
-    const request = this.container.classService.buildRequest(message.author, args);
-    if (!request) {
-      message.reply('I was unable to build your request.');
+    if (!args) {
       return;
     }
-    try {
-      const response = await this.container.classService.register(request);
-      message.reply(response);
-    } catch (e) {
-      message.reply(e);
+
+    const successfulClasses: string[] = [];
+    const invalidClasses: string[] = [];
+    for (const arg of args) {
+      const request = this.container.classService.buildRequest(message.author, [arg]);
+      if (!request) {
+        message.reply('I was unable to build your request.');
```

Comment:
also might not need this.

Reply:
wouldnt want this to be sent n times

Reply:
So check if it failed for each arg, then reply to the user, outside of the for loop, that one or more requests failed?

Reply:
include it in the array of classes that could not be added (no 'success')